Develop

Wumianzhe opened the pull request:
It took a lot of time to sort out branches. Git can be difficult sometimes (all the merge conflicts and unrelated histories). Commit times on github are wrong, actual commits were done multiple weeks ago

Not much done here, mostly groundwork. In my opinion some parts will need a rewrite, but that's not my part of the job.

Wumianzhe committed:
Initial commit

Wumianzhe committed:
Initial commit

Wumianzhe committed:
Plan, Docker image

Can't really do much without framework to put my code in

Wumianzhe committed:
Plan, Docker image

Can't really do much without framework to put my code in

Wumianzhe committed:
manage script

Wumianzhe committed:
manage script

Wumianzhe committed:
manage script v2

added db to ignore, some edits to runner script

Wumianzhe committed:
manage script v2

added db to ignore, some edits to runner script

Wumianzhe committed:
remove files specific to one branch

Should have just cherry-picked but another commit it is

Wumianzhe committed:
remove files specific to one branch

Should have just cherry-picked but another commit it is

popov-p committed:
fixed basic navigation

Wumianzhe committed:
First merge

Project is finally one project and not 3

Wumianzhe committed:
First merge

Project is finally one project and not 3

Wumianzhe committed:
Merge branch 'devel-server' into develop

Wumianzhe committed:
pixi example

Basic use of pixi, unrelated to project as of now.

MaEgV commented on numinteractive/settings.py:
```diff
@@ -0,0 +1,128 @@
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
```
:)

MaEgV commented on ui/ABOUT.html:
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+    <head>
```
Предлагаю сделать base.html, где можно прописать head и други общие теги. Остальные html файлы можно extend-ить от base (механизм должен быть прописан где-то в документации Django)

Wumianzhe committed:
relocation

Still just an example, but project structure is fixed

Wumianzhe committed:
grid, function stub

popov-p committed:
methods

popov-p committed:
Merge branch 'graph' into server

popov-p committed:
stub

Wumianzhe committed:
first draw

Love async. Many races

Wumianzhe committed:
transform

both grid and functions now use Scene data to draw

popov-p committed:
secondary tertiary axis

popov-p committed:
cors and correct response to node

Commit:
add js get response

popov-p committed:
bisection endpoint

popov-p committed:
Merge branch 'graph' of ssh://10.147.159.44:39901/srv/git/ApiProj into graph

Commit:
add canvas to GRAPH.html

Commit:
add id to div and canvas

Commit:
add argd request

Wumianzhe committed:
interfaces and drawing bases

Wumianzhe committed:
Merge branch 'graph' of origin into graph

Upstream updated

popov-p committed:
response formatting

popov-p committed:
valid secant method and serialized data

popov-p committed:
newton endpoint and serialized response data

popov-p committed:
correct epsilon

Commit:
redesign

Commit:
Merge branch 'graph' of ssh://10.147.159.44:39901/srv/git/ApiProj into graph

Wumianzhe committed:
transform changes

Wumianzhe committed:
Merge branch 'origin/graph' into graph

Wumianzhe committed:
semi-usable grid

popov-p committed:
navigation + separate methods file

Wumianzhe committed:
display demonstration

Wumianzhe committed:
Merge remote-tracking branch 'origin/graph' into graph

popov-p committed:
attempt to move front from node to django

Commit:
Add theory template and chenged header

popov-p committed:
grid almost done

Commit:
Add text to INTRO and BISECTION

Commit:
Add NEWTON.html and SECANT.html

Commit:
Add script

Add script in NEWTON.htm and SECANT.htm to transform Latex code to math symbols.

popov-p committed:
text begin

popov-p committed:
Merge branch 'server' into graph

popov-p committed:
django rendering endpoints

Wumianzhe committed:
works in theory

Should work, but I'm getting issues with getting function to draw. I
will revisit that soon if I get any new ideas
